projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1202,13 +1349,16 @@ static function publish_box() {
 						<p>
 							<label>
 								<input type="radio" name="add_to_existing_display" value="true" <?php checked( $add_css ); ?>/>
-								<?php _e( 'Add-on CSS <b>(Recommended)</b>', 'jetpack' ); ?>
+								<?php _e( 'Add-on CSS <b>(Recommended)</b>', 'jetpack' ); // phpcs:ignore WordPress.Security.EscapeOutput.UnsafePrintingFunction ?>
 							</label>
 							<br />
 							<label>
 								<input type="radio" name="add_to_existing_display" value="false" <?php checked( ! $add_css ); ?>/>
-								<?php printf(
-									__( 'Replace <a href="%s">theme\'s CSS</a> <b>(Advanced)</b>', 'jetpack' ),
+								<?php
+
+								printf(
+									// translators: the theme's stylesheet URL.
+									__( 'Replace <a href="%s">theme\'s CSS</a> <b>(Advanced)</b>', 'jetpack' ), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
									wp_kses_post( __( 'Replace <a href="%s">theme\'s CSS</a> <b>(Advanced)</b>', 'jetpack' ) ),
```
(or, again, a custom array)